api/send_submission.php

```diff
@@ -0,0 +1,124 @@
+<?php
+//    ~~Settings~~
+error_reporting(-1);
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+set_include_path('./includes/');
+require_once('mysqli.php');
+require_once('check_file.php');
+require_once('PHPMailer/Exception.php');
+require_once('PHPMailer/PHPMailer.php');
+
+
+//    ~~Declare/recieve data~~
+$name = '';
+$org_name = '';
+$email = '';
+$description = '';
+$name = mysqli_real_escape_string($mysqli, strip_tags(trim($_POST["name"])));
+$org_name = mysqli_real_escape_string($mysqli, strip_tags(trim($_POST["org_name"])));
+$email = mysqli_real_escape_string($mysqli, strip_tags(trim($_POST["email"])));
+$description = mysqli_real_escape_string($mysqli, strip_tags(trim($_POST["description"])));
+$slide_image = $_FILES['slideImage'];
+
+
+//    ~~Validate data~~
+DEFINE('SLIDE_IMAGE_MAX_FILE_SIZE', 4);
+$slide_image_mime_types = array(
+    'png' => 'imge/png',
+    'jpeg' => 'image/jpeg',
+    'jpg' => 'image/jpeg'
+);
+
+//create error message in case a check fails
+if (!isset($result_data)) 
+    $result_data = new stdClass();
+$result_data->status = 'error';
+$result_data->message = '';
+
+//check slide submission image
+$slide_image_check_result = checkFile($slide_image, SLIDE_IMAGE_MAX_FILE_SIZE, $slide_image_mime_types);
+if (!$slide_image_check_result->file_safe) {
+    $result_data->message = $slide_image_check_result->message;
+    echo json_encode($result_data);
+    die();
+}
+
+//check name
+if (!preg_match("/^[\w\ \'\.]{1,256}$/", $name)) {
+    $result_data->message = 'Your name, ' . htmlentities($name) . ', is invalid. Please only use latin characters a-z with an optional '
+        . 'apostrophe or period. Your name is also limited to 128 characters.';
+    echo json_encode($result_data);
+    die();
+}
+
+//check organization name
+if (!preg_match("/^[\w\ \'\.]{1,256}$/", $org_name)) {
+    $result_data->message = 'Your organizaiton name, ' . htmlentities($org_name) . ', is invalid. Please only use latin characters a-z with an optional '
```

Comment:
You will not need the `htmlentities` function when adding the message. This is causes characters to be escaped and will be displayed to the user when showing the error message on the front-end.